### PR TITLE
*: restructure understanding tikv chapter

### PR DESCRIPTION
**Pull request by andylokandy**
### What is changed:

restructured understanding tikv chapter

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -3,26 +3,57 @@
...
+  * [Overview](understanding-tikv/overview/intro.md)
+    * [Raw KV](understanding-tikv/overview/raw-kv.md)
+    * [Transaction KV](understanding-tikv/overview/transaction-kv.md)
+  * [Scalability](understanding-tikv/scalability/intro.md)
+    * [Region](understanding-tikv/scalability/region.md)
+    * [Scheduling](understanding-tikv/scalability/scheduling.md)
+  * [High Availability](understanding-tikv/high-availability/intro.md)
+    * [Byzantine Failure](understanding-tikv/high-availability/byzantine-failure.md)
+    * [Raft](understanding-tikv/high-availability/raft/intro.md)
+      * [raft-rs](understanding-tikv/high-availability/raft/raft-rs.md)
```
Comment (tisonkun) [inline src/SUMMARY.md:32]: May we have another name align with other titles?

**Review by tisonkun — COMMENTED** [review]
Rest LGTM.

File: src/SUMMARY.md (hunk 1 above)
Comment (andylokandy) [reply src/SUMMARY.md:32]: Do you have any suggestions? I'd like to talk about [raft-rs](https://github.com/tikv/raft-rs) in this chapter.

Comment (tisonkun) [reply src/SUMMARY.md:32]: Generally "Raft" is ok, but it conflicts with the chapter title. I don't have a strong feeling before we have any content so if you don't have a good idea, we can leave as is.

**Review by tisonkun — APPROVED** [review]
LGTM. Will merge if no further discussion in hours.

**Pull request merged**